yb-voyager/cmd/common.go

```diff
@@ -1076,3 +1108,26 @@ func updateExportSnapshotDataStatsInPayload(exportDataPayload *callhome.ExportDa
 		}
 	}
 }
+
+func sendCallhomePayloadAtIntervals(ctx context.Context) {
```

Comment:
how/when  is this function closed? 
Is it possible that command has also completed but this func will still be sending `IN-PROGRESS` ?

Reply:
Yes, handled already that now that there is one var keeps track of the complete payload sent `callhomeCompletePayloadSent` and uses it to terminate this function.